streaming/base/compression/compression.py

```diff
@@ -142,7 +142,7 @@ def decompress(self, data: bytes) -> bytes:
 class Zstandard(LevelledCompression):
     """Zstandard compression."""
 
-    extension = 'zstd'
+    extension = 'zst'
```

Comment:
Why changing from `zstd` to `zst` ? From official webpage https://facebook.github.io/zstd/, looks like the abbreviation is `zstd`.

Reply:
- we currently name compression algos by their filename extensions (for example, "gz" not "gzip")
- to my surprise, it turns out "zst" is the official extension, not "zstd" (search for ".zst" in https://github.com/facebook/zstd and elsewhere)